docs/INSTALL.md

```diff
@@ -12,9 +12,11 @@
 
 ## Deploy the Operator
 
-Deploy the operator by running the following command.
+Deploy the operator by running the following command where `<RELEASE_VERSION>` needs to be substituted
+with the desired [release tag](https://github.com/confidential-containers/operator/tags).
+
 ```
-kubectl apply -f https://raw.githubusercontent.com/confidential-containers/operator/main/deploy/deploy.yaml
+kubectl apply -k github.com/confidential-containers/operator/config/release?ref=<RELEASE_VERSION>
```

Comment:
Can we just add a concrete example using v0.2.0 as the RELEASE_VERSION ? It'll be helpful imho ?